.drone.star

```diff
@@ -42,64 +40,88 @@ config = {
+                    "webUISharingInternalGroupsEdgeCases",
+                ],
+                "SharingInternalGroupsToRoot": [
+                    "webUISharingInternalGroupsToRoot",
+                    "webUISharingInternalGroupsToRootEdgeCases",
+                ],
+                "SharingInternalGroupsSharingIndicator": [
+                    "webUISharingInternalGroupsSharingIndicator",
+                    "webUISharingInternalGroupsToRootSharingIndicator",
+                ],
                 "webUISharingInternalUsers": [
                     "webUISharingInternalUsers",
                     "webUISharingInternalUsersCollaborator",
                     "webUISharingInternalUsersShareWithPage",
                 ],
                 "webUISharingInternalUsersBlacklisted": "SharingInternalUsersBlacklisted",
-                "webUISharingInternalUsersExpire": "SharingInternalUsersExpire",
-                "webUISharingInternalUsersExpireToRoot": "SharingInternalUsersExpireToRoot",
-                "webUISharingInternalUsersSharingIndicator": "SharingInternalUsersSharingIndicator",
-                "webUISharingInternalUsersToRoot": "webUISharingInternalUsersRoot1",
-                "webUISharingInternalUsersRoot2": [
+                "SharingInternalUsersExpire": [
+                    "webUISharingInternalUsersExpire",
+                    "webUISharingInternalUsersExpireToRoot",
+                ],
+                "SharingInternalUsersSharingIndicator": [
+                    "webUISharingInternalUsersSharingIndicator",
+                    "webUISharingInternalUsersToRootSharingIndicator",
+                ],
+                "SharingInternalUsersRoot1": [
+                    "webUISharingInternalUsersToRoot",
+                    "webUISharingInternalUsersToRootBlacklisted",
+                ],
+                "SharingInternalUsersRoot2": [
                     "webUISharingInternalUsersToRootCollaborator",
                     "webUISharingInternalUsersToRootPreviews",
                     "webUISharingInternalUsersToRootShareWithPage",
                 ],
-                "webUISharingInternalUsersToRootBlacklisted": "SharingInternalUsersRootBlacklisted",
-                "webUISharingInternalUsersToRootSharingIndicator": "SharingInternalUsersRootSharingIndicator",
                 "webUISharingPermissionsUsers": "SharingPermissionsUsers",
                 "webUISharingPermissionToRoot": "SharingPermissionToRoot",
                 "webUISharingPublicBasic": "SharingPublicBasic",
                 "webUISharingPublicManagement": "SharingPublicManagement",
-                "webUISharingPublicDifferentRoles": "SharingPublicDifferentRoles",
-                "webUISharingPublicExpire": "SharingPublicExpire",
-                "webUITrashbinDelete": "TrashbinDelete",
-                "webUITrashbinFilesFolders": "TrashbinFilesFolders",
-                "webUITrashbinRestore": "TrashbinRestore",
-                "webUIUpload": "Upload",
```

Comment:
Did `webUIUpload` get lost?

Reply:
Its not lost 
https://github.com/owncloud/web/pull/5195/files#diff-6a968d4b7901c8526c8d9a86d1f434737dbfe2b00da21a7d3041ed4e3d58d62eR56

But this approache is prone to accidently losing an entire suite in that way, so I'm trying to make a function that checks all the available test suites and see if any suites are missing in the tests.

Reply:
OK - it moved a long way and I lost track of it.